lib/pytis/util/resolver.py

```
@@ -270,11 +285,69 @@ def clear(self):
     def reload(self):
         """Reload all specification modules and clear all caches."""
+        try:
+            # Python 3.4+
+            from importlib import reload as _reload
+        except ImportError:
+            try:
+                # Python 3.0 to 3.3.
+                from imp import reload as _reload
+            except ImportError:
+                # Python 2 builtin.
+                _reload = reload
```

Ruff (0.12.2)

Undefined name `reload`

(F821)